proto/zip.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+
+import "proto/context.proto";
+
+package zip;
+
+message ZipManifestEntry {
```

Comment:
haha since this is in package zip you can lose the Zip prefix in the "object" types (sorry brandon and I conspired to lead you here into zip.ZipVille)

Reply:
updated--i left Zip in the service name and by extension in the request/response types just to match our rpc naming scheme.